### PR TITLE
Add missing imports from the request builders refactor

### PR DESCRIPTION
**Pull request by starg09**
This fixes the build process after the refactors made in https://github.com/axpendix/tcgone/commit/a551752e545af93d320a3bf047dca816ea7e61e3.

### CODE REVIEW

File: src/tcgwars/logic/util/CardList.groovy [hunk 1]
```diff
@@ -348,14 +348,14 @@ public class CardList extends ArrayList<Card> {
     }
     if (playerType != TcgStatics.bg().currentThreadPlayerType) {
       TcgStatics.block()
+      def ret = TcgStatics.bg().getClient(playerType).selectCard(new CardSelectUIRequestBuilder()
+        .setMinMax(min, max)
+        .setInfo(info)
+        .setCards(cards)
+        .setCustomCardFilter(filter != null ? filter as CustomCardFilter : null)
+        .setCustomPassFilter(passFilter != null ?  passFilter as CustomPassFilter : null)
+        .setShowAsHidden(hidden))
```
Comment (axpendix) [inline src/tcgwars/logic/util/CardList.groovy:357]: I think this should have stayed outside the if clause, don't you think?

Comment (starg09) [reply src/tcgwars/logic/util/CardList.groovy:357]: Honestly didn't look past keeping it as it was, but correcting the casting to the new reference. But yes, I may be wrong but shouldn't this work properly without the ternary if? Will test it out in a bit

Comment (starg09) [reply src/tcgwars/logic/util/CardList.groovy:357]: Yup, seems to be what's causing some Engine errors. Will push a fix on a separate PR in a bit :)

Comment (starg09) [reply src/tcgwars/logic/util/CardList.groovy:357]: Fixed in https://github.com/axpendix/tcgone-engine-contrib/pull/1319

**Pull request merged**